config/containers/logging/configure.md

```diff
@@ -18,6 +18,7 @@ unless you configure it to use a different logging driver.
 
 In addition to using the logging drivers included with Docker, you can also
 implement and use [logging driver plugins](/engine/admin/logging/plugins.md).
+Logging driver plugins are available in Docker 17.05 and higher.
```

Comment:
Do we need to keep this? 17.05 is obsolete for a long time, so don't think worth mentioning

Reply:
I deleted this statement.